src/components/pages/HomePage/index.jsx

```diff
@@ -45,7 +41,9 @@ function getRandomInt() {
 
 // Get randomized image
 let randomBG = backgroundPictures[getRandomInt()];
```

Comment:
This can be a `const`.

```suggestion
const randomBG = backgroundPictures[getRandomInt()];
```